pandas/core/categorical.py

```diff
@@ -361,20 +353,6 @@ def __init__(self, values, categories=None, ordered=None, dtype=None,
 
```

Comment:
there are some comments on the lines above that can be cleaned up (mentioning the checks for the warnings)

Reply:
(done)